packages/notifications/push/amplify_push_notifications/test/amplify_push_noitfications_impl_test.dart

```diff
@@ -460,4 +534,35 @@ void main() {
       },
     );
   });
+
+  test('get launchNotification is consumable', () async {
+    Future.delayed(const Duration(microseconds: 1), () async {
```

Comment:
Shouldn't be needed based off previous convo

Reply:
Yes, but without this, the execution gets stuck at bufferedStream.peek
Keeping this now and will update when I get to the root of it. Will add a todo to make sure it gets done.